Source/version.cs

```diff
@@ -1,4 +1,4 @@
 using System.Reflection;
-// Version 3.4.0, year 2018+4 month 1 day 12
+// Version 3.4.0, year 2018+4, month 1 day 12 as for date January 12th
```

Comment:
Why do we need to change this file? The way without the comma seems to be how we did it before and the date is reflected in the commit date.

Reply:
This can help future releases as there is no mention about how to choose version number, so I thought I can add an example. Do you think instead of this the RELEASE.txt file should hold this information?

Reply:
I agree that some additional explanation can't hurt and think the `RELEASE.txt` is the right place for that. That way we don't have to think about keeping it up to date with every release.